installer/bundle/bundle_skel.sh

```diff
@@ -39,7 +39,7 @@ DSC_PKG=<DSC_PKG>
 SCX_PKG=<SCX_PKG>
 SCRIPT_LEN=<SCRIPT_LEN>
 SCRIPT_LEN_PLUS_ONE=<SCRIPT_LEN+1>
-
+INSTALL_TYPE=<INSTALL_TYPE>
```

Comment:
Move this just after 39, just for consistency. After that, it's just the length stuff.